diff -Naur Signal-Desktop-6.20.2.old/package.json Signal-Desktop-6.20.2/package.json
--- Signal-Desktop-6.20.2.old/package.json	2023-06-12 20:32:41.236237393 +0200
+++ Signal-Desktop-6.20.2/package.json	2023-06-12 20:32:41.572243552 +0200
@@ -317,7 +317,7 @@
     "read-last-lines/mz/thenify-all/thenify": "3.3.1"
   },
   "engines": {
-    "node": "18.14.0"
+    "node": ">=18"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-6.20.2.old/ts/scripts/get-expire-time.ts Signal-Desktop-6.20.2/ts/scripts/get-expire-time.ts
--- Signal-Desktop-6.20.2.old/ts/scripts/get-expire-time.ts	2023-06-12 20:32:41.488242012 +0200
+++ Signal-Desktop-6.20.2/ts/scripts/get-expire-time.ts	2023-06-12 20:32:41.623244487 +0200
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
