diff -Naur Signal-Desktop-6.34.1.old/package.json Signal-Desktop-6.34.1/package.json
--- Signal-Desktop-6.34.1.old/package.json	2023-10-17 15:11:27.468677059 +0200
+++ Signal-Desktop-6.34.1/package.json	2023-10-17 15:11:27.651680370 +0200
@@ -325,7 +325,7 @@
     "read-last-lines/mz/thenify-all/thenify": "3.3.1"
   },
   "engines": {
-    "node": "18.15.0"
+    "node": ">=18"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-6.34.1.old/ts/scripts/get-expire-time.ts Signal-Desktop-6.34.1/ts/scripts/get-expire-time.ts
--- Signal-Desktop-6.34.1.old/ts/scripts/get-expire-time.ts	2023-10-17 15:11:27.550678543 +0200
+++ Signal-Desktop-6.34.1/ts/scripts/get-expire-time.ts	2023-10-17 15:11:27.715681528 +0200
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
