diff -Naur Signal-Desktop-6.31.0.old/package.json Signal-Desktop-6.31.0/package.json
--- Signal-Desktop-6.31.0.old/package.json	2023-09-22 15:26:29.158294276 +0200
+++ Signal-Desktop-6.31.0/package.json	2023-09-22 15:26:29.534301112 +0200
@@ -325,7 +325,7 @@
     "read-last-lines/mz/thenify-all/thenify": "3.3.1"
   },
   "engines": {
-    "node": "18.15.0"
+    "node": ">=18"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-6.31.0.old/ts/scripts/get-expire-time.ts Signal-Desktop-6.31.0/ts/scripts/get-expire-time.ts
--- Signal-Desktop-6.31.0.old/ts/scripts/get-expire-time.ts	2023-09-22 15:26:29.463299821 +0200
+++ Signal-Desktop-6.31.0/ts/scripts/get-expire-time.ts	2023-09-22 15:26:29.534301112 +0200
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
