diff -Naur Signal-Desktop-7.22.2.old/package.json Signal-Desktop-7.22.2/package.json
--- Signal-Desktop-7.22.2.old/package.json	2024-08-29 19:27:26.922945980 +0200
+++ Signal-Desktop-7.22.2/package.json	2024-08-29 19:27:27.155948426 +0200
@@ -370,7 +370,7 @@
     }
   },
   "engines": {
-    "node": "20.15.1"
+    "node": ">= 20"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-7.22.2.old/ts/scripts/get-expire-time.ts Signal-Desktop-7.22.2/ts/scripts/get-expire-time.ts
--- Signal-Desktop-7.22.2.old/ts/scripts/get-expire-time.ts	2024-08-29 19:27:27.042947240 +0200
+++ Signal-Desktop-7.22.2/ts/scripts/get-expire-time.ts	2024-08-29 19:27:27.155948426 +0200
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
