diff -Naur Signal-Desktop-7.11.1.old/package.json Signal-Desktop-7.11.1/package.json
--- Signal-Desktop-7.11.1.old/package.json	2024-06-06 16:25:24.447670455 +0200
+++ Signal-Desktop-7.11.1/package.json	2024-06-06 16:25:24.652672078 +0200
@@ -354,7 +354,7 @@
     "read-last-lines/mz/thenify-all/thenify": "3.3.1"
   },
   "engines": {
-    "node": "20.11.1"
+    "node": ">= 20"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-7.11.1.old/ts/scripts/get-expire-time.ts Signal-Desktop-7.11.1/ts/scripts/get-expire-time.ts
--- Signal-Desktop-7.11.1.old/ts/scripts/get-expire-time.ts	2024-06-06 16:25:24.556671318 +0200
+++ Signal-Desktop-7.11.1/ts/scripts/get-expire-time.ts	2024-06-06 16:25:24.653672086 +0200
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
