diff -Naur Signal-Desktop-5.63.0.old/package.json Signal-Desktop-5.63.0/package.json
--- Signal-Desktop-5.63.0.old/package.json	2022-11-06 11:52:02.868776875 +0100
+++ Signal-Desktop-5.63.0/package.json	2022-11-06 11:52:03.125781406 +0100
@@ -327,7 +327,7 @@
     "sharp/color/color-string": "1.9.0"
   },
   "engines": {
-    "node": "16.15.0"
+    "node": "^16"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-5.63.0.old/ts/scripts/get-expire-time.ts Signal-Desktop-5.63.0/ts/scripts/get-expire-time.ts
--- Signal-Desktop-5.63.0.old/ts/scripts/get-expire-time.ts	2022-11-06 11:52:03.052780119 +0100
+++ Signal-Desktop-5.63.0/ts/scripts/get-expire-time.ts	2022-11-06 11:52:03.125781406 +0100
@@ -8,7 +8,7 @@
 import { DAY } from '../util/durations';
 
 const unixTimestamp = parseInt(
-  execSync('git show -s --format=%ct').toString('utf8'),
+  execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
