diff -Naur Signal-Desktop-6.28.0.old/package.json Signal-Desktop-6.28.0/package.json
--- Signal-Desktop-6.28.0.old/package.json	2023-08-10 07:46:56.172892162 +0200
+++ Signal-Desktop-6.28.0/package.json	2023-08-10 07:46:56.450897203 +0200
@@ -319,7 +319,7 @@
     "read-last-lines/mz/thenify-all/thenify": "3.3.1"
   },
   "engines": {
-    "node": "18.15.0"
+    "node": ">=18"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-6.28.0.old/ts/scripts/get-expire-time.ts Signal-Desktop-6.28.0/ts/scripts/get-expire-time.ts
--- Signal-Desktop-6.28.0.old/ts/scripts/get-expire-time.ts	2023-08-10 07:46:56.326894955 +0200
+++ Signal-Desktop-6.28.0/ts/scripts/get-expire-time.ts	2023-08-10 07:46:56.450897203 +0200
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
