diff -Naur Signal-Desktop-6.48.1.old/package.json Signal-Desktop-6.48.1/package.json
--- Signal-Desktop-6.48.1.old/package.json	2024-02-22 20:24:41.830131496 +0100
+++ Signal-Desktop-6.48.1/package.json	2024-02-22 20:24:41.929133194 +0100
@@ -334,7 +334,7 @@
     "read-last-lines/mz/thenify-all/thenify": "3.3.1"
   },
   "engines": {
-    "node": "18.18.2"
+    "node": ">= 18"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-6.48.1.old/ts/scripts/get-expire-time.ts Signal-Desktop-6.48.1/ts/scripts/get-expire-time.ts
--- Signal-Desktop-6.48.1.old/ts/scripts/get-expire-time.ts	2024-02-22 20:24:41.883132405 +0100
+++ Signal-Desktop-6.48.1/ts/scripts/get-expire-time.ts	2024-02-22 20:24:41.961133743 +0100
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
