diff -Naur Signal-Desktop-6.45.0.old/package.json Signal-Desktop-6.45.0/package.json
--- Signal-Desktop-6.45.0.old/package.json	2024-01-25 11:06:04.048153794 +0100
+++ Signal-Desktop-6.45.0/package.json	2024-01-25 11:06:04.180156007 +0100
@@ -335,7 +335,7 @@
     "read-last-lines/mz/thenify-all/thenify": "3.3.1"
   },
   "engines": {
-    "node": "18.17.1"
+    "node": ">= 18"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-6.45.0.old/ts/scripts/get-expire-time.ts Signal-Desktop-6.45.0/ts/scripts/get-expire-time.ts
--- Signal-Desktop-6.45.0.old/ts/scripts/get-expire-time.ts	2024-01-25 11:06:04.128155136 +0100
+++ Signal-Desktop-6.45.0/ts/scripts/get-expire-time.ts	2024-01-25 11:06:04.248157147 +0100
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
