diff -Naur Signal-Desktop-7.1.1.old/package.json Signal-Desktop-7.1.1/package.json
--- Signal-Desktop-7.1.1.old/package.json	2024-03-07 10:20:05.526103182 +0100
+++ Signal-Desktop-7.1.1/package.json	2024-03-07 10:20:05.655105350 +0100
@@ -337,7 +337,7 @@
     "read-last-lines/mz/thenify-all/thenify": "3.3.1"
   },
   "engines": {
-    "node": "18.18.2"
+    "node": ">= 18"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-7.1.1.old/ts/scripts/get-expire-time.ts Signal-Desktop-7.1.1/ts/scripts/get-expire-time.ts
--- Signal-Desktop-7.1.1.old/ts/scripts/get-expire-time.ts	2024-03-07 10:20:05.585104173 +0100
+++ Signal-Desktop-7.1.1/ts/scripts/get-expire-time.ts	2024-03-07 10:20:05.656105367 +0100
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
