diff -Naur Signal-Desktop-7.0.0.old/package.json Signal-Desktop-7.0.0/package.json
--- Signal-Desktop-7.0.0.old/package.json	2024-03-03 19:21:59.848548798 +0100
+++ Signal-Desktop-7.0.0/package.json	2024-03-03 19:21:59.954550688 +0100
@@ -336,7 +336,7 @@
     "read-last-lines/mz/thenify-all/thenify": "3.3.1"
   },
   "engines": {
-    "node": "18.18.2"
+    "node": ">= 18"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-7.0.0.old/ts/scripts/get-expire-time.ts Signal-Desktop-7.0.0/ts/scripts/get-expire-time.ts
--- Signal-Desktop-7.0.0.old/ts/scripts/get-expire-time.ts	2024-03-03 19:21:59.904549796 +0100
+++ Signal-Desktop-7.0.0/ts/scripts/get-expire-time.ts	2024-03-03 19:21:59.954550688 +0100
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
