diff -Naur Signal-Desktop-6.5.0.old/package.json Signal-Desktop-6.5.0/package.json
--- Signal-Desktop-6.5.0.old/package.json	2023-02-10 08:22:14.747270337 +0100
+++ Signal-Desktop-6.5.0/package.json	2023-02-10 08:22:14.929273434 +0100
@@ -318,7 +318,7 @@
     "read-last-lines/mz/thenify-all/thenify": "3.3.1"
   },
   "engines": {
-    "node": "16.17.1"
+    "node": ">=16"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-6.5.0.old/ts/scripts/get-expire-time.ts Signal-Desktop-6.5.0/ts/scripts/get-expire-time.ts
--- Signal-Desktop-6.5.0.old/ts/scripts/get-expire-time.ts	2023-02-10 08:22:14.862272294 +0100
+++ Signal-Desktop-6.5.0/ts/scripts/get-expire-time.ts	2023-02-10 08:22:15.005274728 +0100
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
