diff -Naur Signal-Desktop-6.42.1.old/package.json Signal-Desktop-6.42.1/package.json
--- Signal-Desktop-6.42.1.old/package.json	2023-12-28 15:13:50.996465471 +0100
+++ Signal-Desktop-6.42.1/package.json	2023-12-28 15:13:51.131467902 +0100
@@ -335,7 +335,7 @@
     "read-last-lines/mz/thenify-all/thenify": "3.3.1"
   },
   "engines": {
-    "node": "18.17.1"
+    "node": ">= 18"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-6.42.1.old/ts/scripts/get-expire-time.ts Signal-Desktop-6.42.1/ts/scripts/get-expire-time.ts
--- Signal-Desktop-6.42.1.old/ts/scripts/get-expire-time.ts	2023-12-28 15:13:51.077466930 +0100
+++ Signal-Desktop-6.42.1/ts/scripts/get-expire-time.ts	2023-12-28 15:13:51.179468766 +0100
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
