diff -Naur Signal-Desktop-6.0.0.old/package.json Signal-Desktop-6.0.0/package.json
--- Signal-Desktop-6.0.0.old/package.json	2022-12-04 09:38:27.246054436 +0100
+++ Signal-Desktop-6.0.0/package.json	2022-12-04 09:38:30.078103639 +0100
@@ -324,7 +324,7 @@
     "sharp/color/color-string": "1.9.0"
   },
   "engines": {
-    "node": "16.15.0"
+    "node": ">=16"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-6.0.0.old/ts/scripts/get-expire-time.ts Signal-Desktop-6.0.0/ts/scripts/get-expire-time.ts
--- Signal-Desktop-6.0.0.old/ts/scripts/get-expire-time.ts	2022-12-04 09:38:27.360056417 +0100
+++ Signal-Desktop-6.0.0/ts/scripts/get-expire-time.ts	2022-12-04 09:38:30.079103656 +0100
@@ -8,7 +8,7 @@
 import { DAY } from '../util/durations';
 
 const unixTimestamp = parseInt(
-  execSync('git show -s --format=%ct').toString('utf8'),
+  execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
