diff -Naur Signal-Desktop-6.43.2.old/package.json Signal-Desktop-6.43.2/package.json
--- Signal-Desktop-6.43.2.old/package.json	2024-01-10 09:48:16.131504729 +0100
+++ Signal-Desktop-6.43.2/package.json	2024-01-10 09:48:16.288507403 +0100
@@ -335,7 +335,7 @@
     "read-last-lines/mz/thenify-all/thenify": "3.3.1"
   },
   "engines": {
-    "node": "18.17.1"
+    "node": ">= 18"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-6.43.2.old/ts/scripts/get-expire-time.ts Signal-Desktop-6.43.2/ts/scripts/get-expire-time.ts
--- Signal-Desktop-6.43.2.old/ts/scripts/get-expire-time.ts	2024-01-10 09:48:16.224506313 +0100
+++ Signal-Desktop-6.43.2/ts/scripts/get-expire-time.ts	2024-01-10 09:48:16.369508782 +0100
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
