diff -Naur Signal-Desktop-7.18.0.old/package.json Signal-Desktop-7.18.0/package.json
--- Signal-Desktop-7.18.0.old/package.json	2024-08-04 20:16:03.844995853 +0200
+++ Signal-Desktop-7.18.0/package.json	2024-08-04 20:16:41.278261501 +0200
@@ -369,7 +369,7 @@
     }
   },
   "engines": {
-    "node": "20.15.1"
+    "node": ">= 20"
   },
   "build": {
     "appId": "org.whispersystems.signal-desktop",
diff -Naur Signal-Desktop-7.18.0.old/ts/scripts/get-expire-time.ts Signal-Desktop-7.18.0/ts/scripts/get-expire-time.ts
--- Signal-Desktop-7.18.0.old/ts/scripts/get-expire-time.ts	2024-08-04 20:16:03.947996584 +0200
+++ Signal-Desktop-7.18.0/ts/scripts/get-expire-time.ts	2024-08-04 20:17:08.158452257 +0200
@@ -9,7 +9,7 @@
 
 const unixTimestamp = parseInt(
   process.env.SOURCE_DATE_EPOCH ||
-    execSync('git show -s --format=%ct').toString('utf8'),
+    execSync('date +%s').toString('utf8'),
   10
 );
 const buildCreation = unixTimestamp * 1000;
